test/x-scrollable.html

```diff
@@ -0,0 +1,67 @@
+<!--
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../iron-scroll-target-behavior.html">
+
+<dom-module id="x-scrollable">
+
+  <style>
+
+    :host {
+      display: block;
+      font: 14px arial;
+    }
+
+    .item {
+      border-bottom: 1px solid #ccc;
+      background-color: white;
+      padding: 20px;
+      width: 200%;
+    }
+
+  </style>
+
+  <template>
+    <template is="dom-repeat" items="[[_getItems(itemCount)]]">
+      <div class="item">[[index]]</div>
+    </template>
+  </template>
+
+</dom-module>
+
+<script>
+  Polymer({
+
+    is: 'x-scrollable',
+
+    properties: {
+
+      itemCount: {
+        type: Number,
+        value: 200
+      }
+
+    },
+
+    behaviors: [
+      Polymer.IronScrollTargetBehavior
+    ],
+
+    _defaultScrollTarget: null,
```

Comment:
If `_defaultScrollTarget` is null, and you don't set `scroll-target` in the test fixtures, what is the scroll target during the tests?